src/components/trace/hat-script-graph.ts

```diff
@@ -503,13 +507,21 @@ export class HatScriptGraph extends LitElement {
     return html`
       <hat-graph-node
         .graphStart=${graphStart}
-        .iconPath=${mdiRoomService}
+        .iconPath=${node.service ? undefined : mdiRoomService}
```

Comment:
Can't we always use slot icon?
```html
<ha-svg-icon slot="icon" .path=${mdiRoomService} ></ha-svg-icon>`

Reply:
rather not, the `hat-graph-node` is an `svg` just adding a path is way cleaner than including an external component inside an `svg`